### PR TITLE
docs: build-instructions: Add steps to build documentation

### PR DESCRIPTION
**Pull request by patrickelectric**
Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

File: docs/build-instructions.md [hunk 1]
```diff
@@ -81,3 +81,18 @@ When installing, you must minimally select all Visual C++ components.
     - `./pingviewer`
 
 > **Tip**: You can use `make -jN` where N is the number of threads in your computer to improve the compiling speed.
+
+## Build documentation
+
+1. Follow the **Source Code** guide.
```
Comment (rafaellehmkuhl) [inline docs/build-instructions.md:87]: What does this line mean, exactly?

**Review by rafaellehmkuhl — APPROVED** [review]

**Comment by rafaellehmkuhl** [comment]
Despite the doubt about what does the first point means, the docs are building fine with the provided instructions here, so approving.

File: docs/build-instructions.md (hunk 1 above)
Comment (patrickelectric) [reply docs/build-instructions.md:87]: Here: https://github.com/bluerobotics/ping-viewer/pull/970/files#diff-c5bc07b84a84f251d50ddb55eec5c5c2645500ded34a5f36e31fcc162c97d803R9

Comment (rafaellehmkuhl) [reply docs/build-instructions.md:87]: The link is pointing nowere.

Comment (patrickelectric) [reply docs/build-instructions.md:87]: Check the line 9 of this same file.

Comment (rafaellehmkuhl) [reply docs/build-instructions.md:87]: Oh, ok.
Maybe "Follow the **Source Code** guide to get the source files." would be more clear.

**Pull request merged**